frontstage/views/passwords.py

```diff
@@ -15,26 +18,29 @@
 # ===== Forgot password =====
 @passwords_bp.route('/forgot-password', methods=['GET', 'POST'])
 def forgot_password():
-
     form = ForgotPasswordForm(request.form)
 
     if request.method == 'POST' and form.validate():
-
         email_address = request.form.get('email_address')
-
-        # TODO do some kind of back end processing to validate the email address, error handling, logging
-        # print('Email address=' + email_address)
-
-        return redirect(url_for('passwords_bp.forgot_password_check_email',
-                                _external=True,
-                                _scheme=getenv('SCHEME', 'http')))
+        post_data = {"email_address": email_address}
+
+        url = app.config['RAS_PARTY_RESET_PASSWORD_REQUEST'].format(app.config['RAS_PARTY_SERVICE'])
```

Comment:
I don't think you introduced this, but I find this approach to constructing a url a bit odd. :)

Reply:
I agree, it's been here since I started working on frontstage though so I've ran with it.
After BRES we can go through the whole thing and update